index.js

```diff
@@ -285,3 +285,8 @@ export const createHandler = async ({
       })
   }
 }
+
+const parseDateStringOrNull = (strOrNull) => {
+  if (strOrNull === undefined || strOrNull === null) return undefined
```

Comment:
slightly confusing: It's called `OrNull` but returns `undefined` and accepts `null` or `undefined`

Reply:
What about `parseDateStringOrUndefined`?

Reply:
I decided to use `parseOptionalDate`